CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Vis Builder] Add field summary popovers ([#2682](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2682))
 - [I18n] Register ru, ru-RU locale ([#2817](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2817))
 - Add yarn opensearch arg to setup plugin dependencies ([#2544](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2544))
+- Add disablePrototypePoisoningProtection configuration ([#2992](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2992))
```

Comment:
Can we make this a little more descriptive?

Reply:
Update!